examples/tutorials/translations/hindi/Part 05 - Welcome to the Sandbox.ipynb

```diff
@@ -0,0 +1,279 @@
+{
```

Comment:
<p>Here translator and editor names seems to be redundant as they are also mentioned below author names.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2909/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thanks Yugandhar!</p><p> </p><p>For some reason, my jupyter notebook doesn't show the 'Authors, ... ' cell. I  had earlier removed it, precisely because the English version doesn't have this information. But apparently, it still somehow lurks in there....  Would you know why?</p>

Reply:
<p>I wonder why it's showing up when it's clearly not in your branch. Anyway I guess it's resolved for now.</p>